### DISCUSSION TITLE
Welcome to Elearning-website Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [ProDaryl/Elearning-website#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kingballer24** [comment]
Hello it's Bryan gatlin working on the course enrollment

**Comment by ghost** [comment]
U should be ashamed of typing any kind of thing here

Le dim. 25 août 2024 à 18:11, Kingballer24 ***@***.***> a
écrit :
